src/ui/PrimitiveBlock.js

```diff
@@ -5,30 +5,27 @@ import {Primitive} from '../parsers/primitives';
 import {RendererContext} from './Context';
 import './PrimitiveBlock.less';
 
-export class RenderedBlockNode extends Component {
+export class BaseRenderedBlockNode extends Component {
   static propTypes = {
     node: PropTypes.instanceOf(ASTNode),
     text: PropTypes.string,
+    renderer: PropTypes.object.isRequired,
   }
 
   static defaultProps = {
     node: null,
     text: '',
-    renderer: null,
   }
 
   componentDidMount() {
     if (this.root) {
       let el = this.root;
       el.firstChild.draggable = true;
-      el.firstChild.addEventListener(
-        'dragstart',
-        this.onDragStart.bind(this)
-      );
+      el.firstChild.addEventListener('dragstart', this.onDragStart);
     }
   }
 
-  onDragStart(event) {
+  onDragStart = event => {
```

Comment:
I meant to ask this in a prior PR: is this change a stylistic one, or is there a difference in meaning here? If so, help a n00b out?

Reply:
There's a difference. It also does `.bind(this)` automatically, so you can write `Class.onDragStart` right away (and don't have to write `Class.onDragStart.bind(Class)`).

Reply:
Whoa! I had no idea. Thanks!